packages/gallery/src/components/helpers/scrollHelper.js

```diff
@@ -217,7 +221,14 @@ function isWithinPaddingHorizontally({
   return res.before < padding && res.after < padding;
 }
 
-function horizontalCssScrollTo(scroller, from, to, duration, isRTL) {
+function horizontalCssScrollTo(
+  scroller,
```

Comment:
I have a stupid rule of thumb that I use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring) for functions with more than 3 arguments. You can adopt it if you want